DEVX_ROADMAP.md

```diff
@@ -1,51 +1,39 @@
 # Developer Experience Roadmap
-(last updated 9/9/2022, next update approximately ~10/9/2022 following a monthly cadence)
+(last updated 10/11/2022, next update approximately ~11/11/2022 following a monthly cadence)
 
 To keep Sui builders up to date with the latest happenings, we are maintaining the following list of developer-facing changes coming in the next ~30 days. While we strive to be accurate, the timing of the landing/release of these features is subject to change. More thorough documentation and references will be available as each feature is released to Devnet. Please continue to monitor Devnet release notes for the source of truth on what is currently deployed--this list is about what's next!
 
 ## Big picture
-* The gateway component will be deprecated. Clients will now communicate directly with full nodes, and full nodes will act as quorum driver (i.e., accept transactions, form a certificate + certificate effects, return the effects to the client)
-* Child objects are getting a big revamp: they no longer need to be passed as explicit transaction inputs, and rich new collection types (e.g., large hash maps with dynamic lookup) will be possible. More details in the Sui Move section.
+* The gateway component will be deprecated soon and we'll give community sufficient heads up once the dates are finalized. With this deprecation, clients will now communicate directly with full nodes, and full nodes will act as quorum driver (i.e., accept transactions, form a certificate + certificate effects, return the effects to the client).
+* Child objects revamp in still WIP. The features eliminates the need to explicitly pass transaction inputs, and enables rich new collection types (e.g., large hash maps with dynamic lookup).
+* We are adding new pay transaction types for generic payments. See below for more details.
 
 ## JSON RPC
-* [Gateway deprecation] `sui_syncAccountState` will be deprecated and removed.
-* [Gateway deprecation] Gateway’s `sui_executeTransaction` method will be replaced by fullnode’s `sui_executeTransaction` method which has an extra parameter that allows users to choose whether a method should block or return immediately.
```

Comment:
Since you used the 'breaking change' notation for other RPC calls later, do you also want to apply that to Gateway deprecationw hich is breaking in that the RPC endpoint is going away?